vector/src/main/java/im/vector/app/features/analytics/extensions/ComposerExt.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.analytics.extensions
+
+import im.vector.app.features.analytics.plan.Composer
+import im.vector.app.features.home.room.detail.composer.MessageComposerViewState
+import im.vector.app.features.home.room.detail.composer.SendMode
+
+fun MessageComposerViewState.toAnalyticsComposer(): Composer =
```

Comment:
tiny formatting comment (ideally the formatter would do this for us), the kotlin convention is typically to move the closing `)` to a new line

```kotlin
fun MessageComposerViewState.toAnalyticsComposer() = Composer(
    inThread = isInThreadTimeline(),
    isEditing = sendMode is SendMode.Edit,
    isReply = sendMode is SendMode.Reply,
    startsThread = startsThread
)
```

same for https://github.com/vector-im/element-android/pull/5378/files#diff-5f35fe0e05f4f8d301892d428e93ca87334bd099a0821f068ea5f5ecfc90adcaR21